Funcky.Async.Test/Extensions/AsyncEnumerableExtensions/ShuffleTest.cs

```diff
@@ -3,17 +3,20 @@
 using Funcky.Async.Extensions;
 using Funcky.Async.Test.TestUtilities;
 using Xunit;
+using Xunit.Sdk;
 
 namespace Funcky.Async.Test.Extensions.AsyncEnumerableExtensions
 {
     public sealed class ShuffleTest
     {
         [Fact]
-        public void AShuffleIsEnumeratedLazily()
+        public void AShuffleIsEnumeratedLazilyAsync()
         {
             var doNotEnumerate = new FailOnEnumerateAsyncSequence<object>();
 
-            _ = doNotEnumerate.Shuffle();
+            var task = doNotEnumerate.Shuffle();
+
+            Assert.Throws<XunitException>(() => task.Result);
```

Comment:
Why don't you use `Assert.ThrowsAsync`?

Reply:
Fixed in : abbcd79266e54af79003374e9e2c870c1ff4f726
I wasn't aware of Assert.ThrowsAsync.